README.md

```diff
@@ -29,7 +29,7 @@ The Aragon Association and representatives of the current Aragon teams will revi
 
```

Comment:
We'll have to think about how we can securely link the Flock proposal and supporting documents to the AGP. My preference would be that all of these documents live on GitHub, so we can reference docs by a link to the commit height as of the time of proposal approval by the AA and from there derive a SHA-256 hash to create a secure, auditable link to the doc.

Another question is whether the teams should create the AGP or the AA on the teams' behalf. It's kind of an awkward dance for them to submit the PR here, get merged by the AA here, then go to the AGP repo and create the AGP PR. As an AGP Editor I would personally prefer that Flock Finance proposals be submitted by the AA, so I do not have to second-guess whether they should be considered or not. (Currently I have to go to the Flock repo and check if the Flock proposal was approved by the AA, to avoid some awkwardness if I merge a Flock AGP that was not approved by the AA and then the AA has to reject the proposal during the AA review stage.)

Reply:
These are good points. I think we should further investigate what's proposed [here](https://github.com/aragon/AGPs/issues/68) to smoothen the process.

Regarding the Flock application process in general. Should we increase flexibility regarding following the process and add something like "not respecting any element in the process could be sufficient for the Aragon Association to reject a given application". An example is if a team submits a proposal 25 days before AGPs are due, which may reveal to be enough time for discussions to happen, we may not want the process to break this application. On the other hand, if a team submits a proposal 15 days before AGPs are due and it looks like discussions with the community are not over and the proposal is far from being ready, we may want to let the AA raise a red flag pointing at the 30 days rule.  This is just a thought I'm having here, there is probably a case stating this could harm the legitimacy of the process and give to much power to the AA.

Reply:
> These are good points. I think we should further investigate what's proposed here to smoothen the process.

Yes we can continue the discussion there in the AGPs repo as I think that is the right place, and we could solve both of the questions I raised if we more tightly integrate Flock with the AGP process.

>  Should we increase flexibility regarding following the process and add something like "not respecting any element in the process could be sufficient for the Aragon Association to reject a given application". An example is if a team submits a proposal 25 days before AGPs are due, which may reveal to be enough time for discussions to happen, we may not want the process to break this application. On the other hand, if a team submits a proposal 15 days before AGPs are due and it looks like discussions with the community are not over and the proposal is far from being ready, we may want to let the AA raise a red flag pointing at the 30 days rule.

@LouisGrx I will leave it up to you how flexible you keep the Flock application guidelines. I personally prefer the hard deadlines because it ensures equal treatment of all applicants, and also ensures that those proposals that are considered have adequate time for review. I don't like the experience where an applicant submits a proposal, there's a bunch of back and forth, and then the proposal is rejected anyways because there wasn't enough time to discuss.

But if you do decide to leave it flexible, rather than flagging the 30-day rule in such a case (which isn't really a rule since it is flexible) you could just rely on the fact that the proposal is simply not ready to be voted on.

So, you could say:

> We strongly encourage Flock applicants to present their proposal at least 30 days before AGPs for the next vote are due...

And when the time comes, it's up to your judgement to decide if a proposal should make it onto the final ballot or not. The standard for approval is then not whether the application was turned in on time, but whether it is a good application at the time AGPs are due. If it is not, and the reason is because the applicant submitted their proposal too late, well that is their own fault and they should have abided by the strong encouragement to submit earlier. But again I don't really like this UX from an applicant or a reviewer's perspective.

Reply:
Indeed, I feel that hard deadlines are clearer and less tricky to deal with. One good thing would be to make a bit more noise around the Flock application deadline and precise the dates for each voting cycle.